crates/router/src/routes/app.rs

```diff
@@ -1170,7 +1170,12 @@ impl Refunds {
     pub fn server(state: AppState) -> Scope {
         let mut route = web::scope("/v2/refunds").app_data(web::Data::new(state));
 
-        route = route.service(web::resource("").route(web::post().to(refunds::refunds_create)));
+        route = route
+            .service(web::resource("").route(web::post().to(refunds::refunds_create)))
+            .service(
+                web::resource("/{id}/update_metadata")
```

Comment:
reason can also be updated through this endpoint. In future we may add other updatable fields as well
```suggestion
                web::resource("/{id}/update")
```